src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -346,6 +347,7 @@ def _int_to_bn(self, num, bn=None):
         ownership of the object). Be sure to register it for GC if it will
         be discarded after use.
         """
+        assert (bn is None) or (bn is not None and bn != self._ffi.NULL)
```

Comment:
Right, I don't think the line I proposed is semantically different from what you have, just a bit shorter.